spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/vision/CloudVisionProperties.java

```diff
@@ -35,7 +35,27 @@
 	@NestedConfigurationProperty
 	private final Credentials credentials = new Credentials(GcpScope.CLOUD_VISION.getUrl());
 
+	private int executorThreadsCount = 1;
+
+	private int jsonOutputBatchSize = 20;
```

Comment:
I think there is a javadoc requirement for these? Can you look at other `...Properties` classes for an example?

Reply:
Gotcha, will add these with the docs. As reminder to add javadoc to properties, I created https://github.com/spring-cloud/spring-cloud-gcp/issues/1548 to track.